lte/gateway/python/integ_tests/defs.mk

```diff
@@ -276,6 +276,13 @@ s1aptests/test_restore_config_after_non_sanity.py
 #s1aptests/test_ipv6_non_nat_dp_dl_udp.py
 #---------------
 
+# TODO: Add ipv6 tests to integ test suite
```

Comment:
Why are we not adding these test cases as part of sanity now? Are these test cases flaky for connection refused? If yes we can add these test cases in the flaky test cases list above after segment from line nos. 273 to 276, otherwise the 2 different segments will create confusion.

Reply:
I have tested them individually so far. I still have to add them to sanity suite and test multiple times before concluding them as stable/flaky which is going to take some time. Also some tests in the sanity suite like test_attach_detach_multiple_ip_blocks_mobilityd_restart.py and test_attach_detach_setsessionrules_tcp_data.py are randomly failing. Hence, I will create a new PR to add these tests into integ_test suite.